go.mod

```diff
@@ -3,68 +3,77 @@ module github.com/cybozu-go/neco-gcp
 go 1.21
```

Comment:
Cloud Functions does not support go 1.22 yet. So, I keep the minimal Golang version as 1.21.
https://cloud.google.com/functions/docs/runtime-support?hl=ja#go